code/modules/mob/living/carbon/human/ai/human_ai_interaction.dm

```diff
@@ -4,56 +4,72 @@
 	return INFINITY
 
 /atom/proc/human_ai_act(mob/living/carbon/human/ai_human, datum/human_ai_brain/brain)
+	brain.unholster_any_weapon()
+	ai_human.a_intent = INTENT_HARM
 	ai_human.do_click(src, "", list())
```

Comment:
why change this?

Reply:
Now that i think of it, it's not that smart to place intent preference in /atom/ parent proc - because the sole purpose of those lines were to make sure we punch mobs instead of disarming them and etc.
Still, this logic doesn't interfere with most atom interactions, so i think we can leave it be? Maybe the tests will tell otherwise